.github/workflows/lint.yaml

```diff
@@ -0,0 +1,28 @@
+name: Lint
+
+on:
+  push:
+    branches-ignore:
+      - main
+
+jobs:
+  commitlint:
+    runs-on: ubuntu-20.04
+    timeout-minutes: 5
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Set up Node
+        uses: actions/setup-node@v2
+        with:
+          node-version: 14
+
+      - name: Fetch Git history
+        run: git fetch --no-tags --depth=50 origin main
+
+      - name: Install dependencies
+        run: yarn install --frozen-lockfile
+
+      - name: Check Format
+        run: yarn format
```

Comment:
Let's be more specific here. `yarn format` looks more like fixing than checking so I believe `yarn format:check` will serve better here.

For the same reason, I found the shorthand `yarn format` rather confusing but that's another discussion that we can have anytime later.